src/auth/auth.service.ts

```diff
@@ -15,44 +21,67 @@ export class AuthService {
   async login(payload: LoginUserDto) {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const { password: pass, ...input } = payload;
```

Comment:
If not using the destructured password, [don't create a variable for it](https://stackoverflow.com/a/53443378/7145074).
```suggestion
    const input = (({ password, ...rest }) => rest)(user);
```